### PR TITLE
Fixed a few small issues in various js files. Fixes #251 and hopefully #...

### LINKED ISSUE #251: 404 Error on some pages
[Error] Failed to load resource: the server responded with a status of 404 (Not Found) (inline-multiplechoice.js, line 0)

### PR DESCRIPTION
**Pull request by peterasujan**
...244.

### CODE REVIEW

File: admin/bjc-curriculum.js [hunk 1]
```diff
@@ -18,14 +18,16 @@ bjc.secondarySetUp = function() {
...
     }
     var titleText = document.title;
     if (titleText && $(".header").length == 0) {
 	$('<div class="header"></div>').prependTo($("#full")).html(titleText);
-	if (getParameterByName("title") != "") {
-	    $(".header").html(getParameterByName("title"));
-	}
+
+	// I don't think this does anything. If nothing breaks I'll remove it.
+	/* if (getParameterByName("title") != "") {
```
Comment (cycomachead) [inline admin/bjc-curriculum.js:28]: Doesn't this set the text of the top header based on `<title>` in `<head>`?

Comment (cycomachead) [inline admin/bjc-curriculum.js:28]: Oh wait, it's using the URL parameter...hmm. I don't think we do that ever...

File: admin/bjc-loader.js [hunk 2]
```diff
@@ -37,7 +37,7 @@ return (( typeof jQuery == 'function') &&
 ////////// stage 1
 // all these scripts depend on jquery, loaded in stage 0
 // all quiz item types should get loaded here
-bjc.paths.scripts[1] = ["/admin/jquery-ui.1.10.2.min.js", "/admin/quiz/multiplechoice.js", "/admin/quiz/inline-multiplechoice.js"];
```
Comment (cycomachead) [inline admin/bjc-loader.js:40]: Yeah, I made this fix too.

**Comment by cycomachead** [comment]
I think this is fine. It'll cause merge conflicts, but that's OK.

File: admin/bjc-curriculum.js (hunk 1 above)
Comment (xtitter) [inline admin/bjc-curriculum.js:28]: I have a dim memory about that, where we would change the title of a page depending on how it was used.  (e.g., if we use the same page at two different points, maybe this was a way to have the title be different slightly?).  

Eh, we never use it.

**Pull request merged**